README.md

```diff
@@ -148,6 +148,7 @@ For local development and familiarizing yourself with Storm/Mesos, please see th
 * `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
+* `mesos.constraints`: Constrain slave nodes which bootstrap supervisor based on mesos-attributes.  
```

Comment:
I would also like to know how other Mesos Frameworks handle this type of thing, have you mimicked this based off of some other Framework's behavior?